docs/default_base_image.md

```diff
@@ -8,6 +8,32 @@ For WAR projects, the default is the official [`jetty`](https://hub.docker.com/_
 
 Note that Jib's default choice for AdoptOpenJDK and Jetty does not imply any endorsement to them. In fact, for strong reproducibility (which also results in better performance and efficiency), we always recommend configuring [`jib.from.image`](https://github.com/GoogleContainerTools/jib/tree/master/jib-gradle-plugin#from-closure) (Gradle) or [`<from><image>`](https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#from-object) (Maven) to pin to a specific base image using a digest (or at least a tag). And while doing so, you should do your due diligence to figure out which base image will work best for you.
 
+### Docker Hub Download Rate Limit
+
+Docker Hub enforces download rate limit. Because Jib pulls base images from Docker Hub, you can occasionally lead to the following error:
+```
+    > com.google.cloud.tools.jib.plugins.common.BuildStepsExecutionException: 429 Too Many Requests
+      GET https://registry-1.docker.io/v2/library/adoptopenjdk/manifests/sha256:9db4a57b38b6d928761ec9c5a250677d306095df0f6a6bdd8936628e033b9f1a
+      {
+        "errors": [
+          {
+            "code": "TOOMANYREQUESTS",
+            "message": "You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit"
+          }
+        ]
+      }
+```
+Note that, even after Jib fully cached a base image, Jib still connects to Docker Hub to check the imaage every time it runs (unless you pin to a specific base image using a SHA digest). This is to check if the cached base image is up-to-date.
+
+Some options:
+* Configure a registry mirror.
+* Prevent Jib from accessing Docker Hub (after Jib cached a base image locally).
+  * Pin to a specific base image using a SHA digest (for example, `jib.from.image=adoptopenjdk:11-jre@sha256:...`).
+  * Do offline building.
+* Retry with increasing backoffs.
+
+See this [FAQ](https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#i-am-hitting-docker-hub-rate-limits-how-can-i-configure-registry-mirrors) for more details about the options.
```

Comment:
This FAQ only talks about registry mirrors. We may not needed if we mention it above?

Reply:
Oops my bad, I didn't see the comment above.